modules/core/LightSpot.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include "ILight.h"
+#include <corecrt_math_defines.h>
```

Comment:
Interesting :-)
Perhaps it makes sense to make use of it for the whole library globally. I will take care about this.

Reply:
Ok